ee/tabby-ui/app/(home)/page.tsx

```diff
@@ -46,6 +47,10 @@ function MainPanel() {
     query: contextInfoQuery
   })
   const scrollY = useStore(useScrollStore, state => state.homePage)
+  const [selectedModel, setSelectedModel] = useLocalStorage<string>(
```

Comment:
please handle situation if current persisted selectedModel doesn't exist in `/v1beta/models`